backend/config/corpora-api.yml

```diff
@@ -515,6 +453,54 @@ paths:
         "404":
           $ref: "#/components/responses/404"
 
+  /dp/v1/collections/{collection_uuid}/upload/link:
```

Comment:
discussion: I think I didn't think of this earlier 😆 But does it make sense to change the URL to `/dp/v1/collections/{collection_uuid}/links`, since that feels closer to API best practice of using plurals?

Another thing I'm thinking is, using plurals and perhaps accepting an array of links would be future proof, since we want to support selecting multiple dropbox links in one go or something

But this is definitely not a blocker!

Reply:
How about `/dp/v1/collections/{collection_uuid}/upload-links`?

If we accepted and array, we would need to return a list of dataset_uuids and how it maps to each link. Things we can consider in the future. I recommend writing up a ticket for this future work.

Reply:
`/dp/v1/collections/{collection_uuid}/upload-links` looks great! Thank you!

Yeah multiple uploads will be for the future us to worry about 😆